test/helpers/IntegrationHelper.d.ts

```diff
@@ -0,0 +1,87 @@
+type BaseCard = import('../../server/game/core/card/BaseCard').BaseCard;
```

Comment:
Just curious, what's the `.d.ts` mean?

Reply:
definition file, no code, only typings. It's primarily used to pair with js files rather than turn them into TS files.